src/utils/work/fetchWork.js

```diff
@@ -32,6 +33,8 @@ const fetchGeneExpressionWorkWithoutLocalCache = async (
     getState,
   );
 
+  onETagGenerated(ETag);
```

Comment:
`fetchWork` is intended to be generic and `onETagGenerated` seems to be something specific to `loadDownsampledGeneExpression`. Is it necessary to add a new parameter in `dispatchWorkRequest`? Can we reuse the `extras`? Or something else? I am just wary of having another exception in the code that does apply only to a specific case

Reply:
It is something that was added a pretty long time ago already, now I just added it here because it was missing, I'll check if I can actually remove it from here again

Reply:
Ok, just checked. I had added it because it was necessary for `loadMarkerGenes`. Now it is necessary for `loadDownsampledGeneExpression` and that's why it's here too now.

You can see it is already coming from `optionals`, so it is already in an object. 
`extras` can't be used for that because anything you add to extras goes into the hash to generate the ETag, and this function is just a callback.

Reply:
It is indeed coming from optionals. I looked at the code and from what you are saying, it sounds indeed fine to have it in `fetchGeneExpressionWorkWithoutLocalCache`